app.json

```diff
@@ -10,7 +10,12 @@
       "bundleIdentifier": "com.kiwi.rn"
     },
     "android": {
-      "package": "com.kiwi.rn"
+      "package": "com.kiwi.rn",
+      "config": {
+        "googleMaps": {
+          "apiKey": "** GOOGLE MAPS API KEY HERE **"
```

Comment:
I am curious - how does it work if you do not have Google Maps installed on your device?

Reply:
it should use google maps tiles, not google maps app itself imho